script/c100331100.lua

```diff
@@ -84,4 +84,4 @@ function c100331100.desop(e,tp,eg,ep,ev,re,r,rp)
 			Duel.SendtoGrave(sg,REASON_EFFECT)
 		end
 	end
-end
\ No newline at end of file
+end
```

Comment:
usually GitHub doesn't show such files in the request
maybe it's a removed extra space
those are only 2 and won't cause anything
it doesn't deserve to redo the whole thing.